data_structures/src/superblock.rs

```diff
@@ -164,6 +183,53 @@ impl SuperBlockState {
         }
     }
 
+    /// Return true if the local superblock has the majority of votes
+    pub fn has_consensus(&self) -> SuperBlockConsensus {
+        log::info!("Superblock votes: {:?}", self.votes_on_each_superblock);
+        log::info!("Previous ars: {:?}", self.previous_ars_identities);
+        // If previous_ars_identities is None, this is the first superblock. The first superblock
+        // is the one with index 0 and genesis hash. These are consensus constants and we do not
+        // need any votes to determine that that is the most voted superblock.
+        if self.previous_ars_identities.is_none() {
+            return SuperBlockConsensus::SameAsLocal;
+        }
+        let identities_that_can_vote = self.previous_ars_identities.as_ref().unwrap().len();
```

Comment:
```suggestion
        let identities_that_can_vote = self.current_signing_committee.as_ref().unwrap().len();
```

Reply:
At the moment this change doesn`t make any difference, since the current_signing_committee is set to be equal the previous_ars_identities, but it will be useful when the members that vote superblocks are a subset of the ARS.